src/org/thoughtcrime/securesms/OnSwipeTouchListener.java

```diff
@@ -0,0 +1,58 @@
+package org.thoughtcrime.securesms;
+import android.content.Context;
+import android.view.GestureDetector;
+import android.view.GestureDetector.SimpleOnGestureListener;
+import android.view.MotionEvent;
+import android.view.View;
+import android.view.View.OnTouchListener;
+
+/**
+ * Created by shawnelbaz on 2018-02-09.
```

Comment:
no comments

Reply:
same, this is redundant class, we might need it in the future tho